### PR TITLE
Add date of observations to charts

### PR DESCRIPTION
**Pull request by chejennifer**
If the chart is not a line chart and does not have time as its axis, put in brackets in the title what date the data in the chart is from
Screenshot 2020-09-15 at 10 23 47 AM

### CODE REVIEW

File: static/css/place/place.scss [hunk 1]
```diff
@@ -171,6 +171,7 @@ $sidebar-top-padding: 132px;  // Keep in sync with place.ts: Y_SCROLL_LIMIT
 
 .chart-container h4 .sub-title {
   color: #999;
+  padding-left: 2px;
```
Comment (beets) [inline static/css/place/place.scss:174]: Can we use .25rem instead (that way the space will scale with the base font size)

File: static/js/place/place_template.tsx [hunk 2]
```diff
@@ -512,6 +512,7 @@ interface ChartStateType {
   dataPoints?: DataPoint[];
   dataGroups?: DataGroup[];
   elemWidth: number;
+  dateSelected?: string;
```
Comment (beets) [inline static/js/place/place_template.tsx:515]: since titleString doesn't seem to be used, let's remove it from the class

**Review by beets — CHANGES_REQUESTED** [review]
thanks! just a couple of small changes

File: static/js/place/place_template.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/place/place_template.tsx:515]: I don't see titleString, do you mean titleSuffix?

Comment (beets) [reply static/js/place/place_template.tsx:515]: oh yes, titlesuffix

**Review by beets — APPROVED** [review]
great, thanks!

**Pull request merged**